config/config.go

```diff
@@ -171,8 +177,27 @@ func Get() (Config, []FutureLog) {
 	}
 
 	var errs []FutureLog
-	config.ExternalIPV4, config.ExternalIPV6, errs = validateExternalIP(config.ExternalIP)
-	logs = append(logs, errs...)
+
+	if len(config.TurnExternalIP) > 0 {
```

Comment:
Could you give me a hint what I'm doing wrong? I'm starting coturn(v4.5.2) with:
```
turnserver -n --log-file=stdout -v --external-ip=192.168.178.2 --use-auth-secret --static-auth-secret=W85lGGbslu9iHFeUHdH3NfoLp3voXKo8lI7RoUidOV3wiJmk7aZ3mylONrprWK3M
```
My screego config includes:
```
SCREEGO_TURN_EXTERNAL_IP=192.168.178.2
SCREEGO_TURN_EXTERNAL_SECRET=W85lGGbslu9iHFeUHdH3NfoLp3voXKo8lI7RoUidOV3wiJmk7aZ3mylONrprWK3M
```
Connecting with stun works fine, but with forceTurn=true it fails:
```
23: : session 012000000000000001: realm <> user <>: incoming packet message processed, error 401: Unauthorized
```

Reply:
Hmmmm 🤔 

Just comparing with my config, I have coturn running in docker so I have to use `external-ip=<public>/<private>` format (one for ipv4 pair and one for ipv6 pair), I also have `realm` configured to a domain name (not sure if that's useful, I put IP addr in `SCREEGO_TURN_EXTERNAL_IP`), some TLS params (which I guess are not relevant either) and `fingerprint` option (again not sure if it's relevant).

I run the same coturn version as you.

Reply:
Okay, I got it working. Apparently the default realm is required. Also, something in the browser is cached, so, if you connect to coturn (without defined realm) and then define the realm and try to reconnect, it'll still fail. A browser restart is needed (:.